src/www/ui_tests/api/Controllers/UserControllerTest.php

```diff
@@ -233,15 +240,21 @@ public function testGetCurrentUser()
   {
     $userId = 2;
     $user = $this->getUsers([$userId]);
-    $user[0]["default_group"] = "fossy";
+
     $this->restHelper->shouldReceive('getUserId')->andReturn($userId);
-    $this->dbHelper->shouldReceive('getUsers')->withArgs([$userId])
+     $this->dbHelper->shouldReceive('getUsers')->withArgs([$userId])
```

Comment:
Please fix indentation.
```suggestion
    $this->dbHelper->shouldReceive('getUsers')->withArgs([$userId])
```

Reply:
Reminder @dushimsam